recipes/starshot_event_content_type/config/node.type.event.yml

```diff
@@ -2,8 +2,12 @@ langcode: en
 status: true
 dependencies:
   module:
+    - menu_ui
     - scheduler
 third_party_settings:
+  menu_ui:
+    available_menus: {  }
+    parent: ''
```

Comment:
I'd rather we remove all the `menu_ui` stuff. It seems unlikely that people will want to list _individual events_ in any of the menus.

Reply:
The change here is actually to exclude it from showing any available menus by default. Otherwise, new events have the option to be added to the main navigation menu.

Reply:
Gotcha - I think it's okay to exclude it anyway. I'd rather not incur the cost of installing an extra module (menu_ui) just to cover this third-party setting. :)

Reply:
Removed.